src/components/Popover/Popover.module.scss

```diff
@@ -68,35 +68,35 @@
 
     // Arrows
     .isRootAtTop > .arrow {
-        top: 100%;
+        top: calc(100% - #{theme.$arrow-safe-rendering-overlap + theme.$arrow-gap});
```

Comment:
We only want to shift the arrow by `$arrow-gap`. Otherwise the resulting gap is 2 px (with current theme values):

<img width="846" alt="obrazek" src="https://github.com/user-attachments/assets/edabbbe2-c661-4aac-a460-97d4e13dbe18">
⬆️ 2px gap here

```suggestion
        top: calc(100% - #{theme.$arrow-gap});
```